DxDispatch/src/dxdispatch/Device.cpp

```diff
@@ -348,7 +355,13 @@ void Device::RecordDispatch(const char* name, uint32_t threadGroupX, uint32_t th
     {
         m_commandList->Dispatch(threadGroupX, threadGroupY, threadGroupZ);
         if (!m_postDispatchBarriers.empty())
-            m_commandList->ResourceBarrier(m_postDispatchBarriers.size(), m_postDispatchBarriers.data());
+        {
+            if(m_postDispatchBarriers.size() > std::numeric_limits<uint32_t>::max())
```

Comment:
```suggestion
            if (m_postDispatchBarriers.size() > std::numeric_limits<uint32_t>::max())
```

And all the other places too...

Reply:
Seems there are still 7 places.